typescript/sdk/src/deploy/EvmModuleDeployer.ts

```diff
@@ -254,6 +254,7 @@ export class EvmModuleDeployer<Factories extends HyperlaneFactories> {
     threshold?: number;
     multiProvider: MultiProvider;
   }): Promise<Address> {
+    values = values.sort();
```

Comment:
would copy what the ism factory does, as `.sort()` is in-place
```suggestion
    values = [...values].sort();
```

Reply:
One of the worst design decisions in Javascript's history. I see people make this mistake very often

Reply:
Sloppy of me, especially given that I'd done it the right way in a previous commit